pkg/tasks/tasks.go

```diff
@@ -116,20 +116,36 @@ func WithFullInstall(t Tasks) Tasks {
 			{Fn: saveKubeconfig, ErrMsg: "failed to save kubeconfig to the local machine"},
 			{Fn: restartKubeAPIServer, ErrMsg: "failed to restart unhealthy kube-apiserver"},
 		}...).
-		append(kubernetesResources()...).
+		append(WithResources(nil)...).
 		append(
 			Task{Fn: createMachineDeployments, ErrMsg: "failed to create worker machines"},
 		)
 }
 
-func WithRefreshResources(t Tasks) Tasks {
+func WithResources(t Tasks) Tasks {
 	return t.append(
 		Tasks{
+			{
+				Fn:         renewCerts,
```

Comment:
The kubeconfig will be invalidated if the certificates are renewed, so we should download the new kubeconfig as well.

Reply:
Now we are refreshing downloaded kubeconfig every time.